scripts/ci/run-benchmark

```diff
@@ -0,0 +1,127 @@
+            (file_size, large_file_dir))
+
+
+def main(args):
+    initialize_files(args.num_files, args.large_file_size)
+    run_id = generate_run_id()
+    results_dir = os.path.join(WORKDIR, 'results', run_id)
+    os.makedirs(results_dir)
+    benchmark(args.bucket, results_dir, args.num_iterations)
+    print("RUN ID: " + run_id)
+
+
+def benchmark(bucket, results_dir, num_iterations=1):
+    perf_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    perf_dir = os.path.join(perf_dir, 'performance')
+
+    s3_location = bucket + '/' + MANY_FILES_DIR
+    local_dir = os.path.join(WORKDIR, MANY_FILES_DIR)
+    try:
+        # 10k upload
+        results = os.path.join(results_dir, 'upload-10k-small')
+        os.makedirs(results)
+        benchmark_cp = os.path.join(perf_dir, 'benchmark-cp')
+        run(benchmark_cp + ' --recursive --num-iterations %s '
+            '--source %s --dest %s --result-dir %s --no-cleanup' % (
+                num_iterations, local_dir, s3_location, results))
+
+        # 10k download
+        results = os.path.join(results_dir, 'download-10k-small')
+        os.makedirs(results)
+        run(benchmark_cp + ' --recursive --num-iterations %s '
+            '--source %s --dest %s --result-dir %s' % (
+                num_iterations, s3_location, local_dir, results))
+
+        # 10k rm
+        results = os.path.join(results_dir, 'delete-10k-small')
+        os.makedirs(results)
+        benchmark_rm = os.path.join(perf_dir, 'benchmark-rm')
+        run(benchmark_rm + ' --recursive --num-iterations %s '
+            '--target %s --result-dir %s' % (
+                num_iterations, s3_location, results))
+    except Exception:
+        # Just in case
+        run('aws s3 rm --recursive ' + s3_location)
+        raise
+
+    s3_location = bucket + '/' + LARGE_FILE_DIR
+    local_dir = os.path.join(WORKDIR, LARGE_FILE_DIR)
+    try:
+        # 10gb upload
+        results = os.path.join(results_dir, 'upload-10gb')
+        os.makedirs(results)
+        run(benchmark_cp + ' --recursive --num-iterations %s '
+            '--source %s --dest %s --result-dir %s --no-cleanup' % (
+                num_iterations, local_dir, s3_location, results))
+
+        # 10gb download
+        results = os.path.join(results_dir, 'download-10gb')
+        os.makedirs(results)
+        run(benchmark_cp + ' --recursive --num-iterations %s '
+            '--source %s --dest %s --result-dir %s' % (
+                num_iterations, s3_location, local_dir, results))
+    finally:
+        # Not benchmarking a single rm call since it's just a single call
+        run('aws s3 rm --recursive ' + s3_location)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-n', '--num-iterations', type=int, default=10,
+        help='The number of times to run each test.'
+    )
+    parser.add_argument(
+        '-b', '--bucket', default=TEST_BUCKET,
```

Comment:
One small thing I noticed.  The arg is called "bucket" but also requires the `s3://` prefix.  I like having this just be the bucket name and have the script inject the `s3://` prefix where needed.  Same thing with the env var `TEST_BUCKET`